app/src/main/java/org/transitclock/api/data/ApiServiceIdResponse.java

```diff
@@ -0,0 +1,30 @@
+/* (C)2023 */
+package org.transitclock.api.data;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Data;
+
+/**
+ * For outputting simple list of unsorted service IDs with lists of sorted block IDs
+ */
+@Data
+public class ApiServiceIdResponse {
+
+    @JsonProperty("serviceIds")
+    private List<ApiServiceId> apiServiceIds;
+
+    /**
+     * Creates the API unsorted version of list of IDs.
+     *
+     * @param serviceIds
+     */
+    public ApiServiceIdResponse(Map<String, List<String>> serviceIds) {
+        apiServiceIds = new ArrayList<>();
+        serviceIds.forEach((key, list) -> apiServiceIds
```

Comment:
just stream over the received serviceIds and map  them to the list needed to have a cleaner impl

Reply:
Well done. @vesavlad  Please check it.